### PR TITLE
Configure workers to use local DNS server

### PR DESCRIPTION
**Pull request by cybertron**

### CODE REVIEW

**Comment by cybertron** [comment]
@celebdor Is this what you were looking for?

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/626/

File: assets/templates/99_worker-dhclient-dns.yaml [hunk 1]
```diff
@@ -15,9 +15,9 @@ spec:
           verification: {}
         filesystem: root
         mode: 0664
-        path: /etc/dhcp/dhclient.conf
+        path: /etc/dhcp/dhclient.conf.template
```
Comment (cybertron) [inline assets/templates/99_worker-dhclient-dns.yaml]: Also needed this change for the file to be written correctly.

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/630/

**Pull request merged**